ilc/server/i18n.js

```diff
@@ -59,21 +59,6 @@ const onRequestFactory = (i18nConfig, i18nParamsDetectionPlugin) => async (req,
     }
 }
 
-/**
```

Comment:
I have decided to move it to common because it is used on both server and client sides